hub-templates/base-hub/values.yaml

```diff
@@ -128,6 +134,8 @@ jupyterhub:
   auth:
     type: dummy
   hub:
+    nodeSelector:
+      hub.jupyter.org/pool-name: core-pool
```

Comment:
The configuration scheduling.corePods.nodeAffinity.matchNodePurpose=require will make all core pods of the jupyterhub chart have a required node affinity toward the node label `hub.jupyter.org/node-purpose=core`.

By using hub.jupyter.org/pool-name it seems like there is a predefined meaning of that label and can cause some confusion, it isn't having a predefined meaning right? If it is our meaning, we should do something like 2i2c.org/pool-name instead I think.

Reply:
Great catch, @consideRatio! I opened https://github.com/2i2c-org/pilot-hubs/issues/136 to track this, since changing the labels will require rotating node pools.